Assets/LeapMotionModules/InteractionEngine/Scripts/InteractionBehaviour.cs

```diff
@@ -6,6 +6,7 @@
 using UnityEngine.Assertions;
 using Leap.Unity.Interaction.CApi;
 using LeapInternal;
+using UnityEditor;
```

Comment:
Make sure to wrap this in an ifdef or else unity will yell at you about missing namespaces when you try to build